integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -574,11 +574,34 @@ export class CdcJsService implements OnDestroy {
           if (currentEmail !== userDetails.uid) {
             this.logoutUser();
           }
+          this.handleProfileUpdateToastMessage(response);
         });
       });
     }
   }
 
+  /**
+   * handle toast message on profile update
+   * @param response
+   *
+   */
+  handleProfileUpdateToastMessage(response?: any) {
```

Comment:
Can we change the method name from `handleProfileUpdateToastMessage` to `handleProfileUpdateResponse` ? because 'toast' is not usually used in any libraries in spa.

Reply:
Done